packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -305,6 +307,22 @@ export class DeltaManager
         return this._reconnectMode;
     }
 
+    public async connectToStorage(): Promise<IDocumentStorageService> {
+        const service = this.serviceProvider();
```

Comment:
maybe add assert that
this.storageService === undefined
I.e. allow for now to call it once?
Or return this.storageService if you already have one?

Reply:
if already created, then returned the same storage service.